test/e2e/helpers.js

```diff
@@ -545,13 +562,72 @@ async function waitForAccountRendered(driver) {
     '[data-testid="eth-overview__primary-currency"]',
   );
 }
+const WINDOW_TITLES = Object.freeze({
+  ExtensionInFullScreenView: 'MetaMask',
+  TestDApp: 'E2E Test Dapp',
+  Notification: 'MetaMask Notification',
+  ServiceWorkerSettings: 'Inspect with Chrome Developer Tools',
+  InstalledExtensions: 'Extensions',
+});
 
 const logInWithBalanceValidation = async (driver, ganacheServer) => {
   await driver.fill('#password', 'correct horse battery staple');
   await driver.press('#password', driver.Key.ENTER);
   await assertAccountBalanceForDOM(driver, ganacheServer);
 };
 
+async function unlockWallet(driver, walletPassword) {
```

Comment:
You can use `login` from helper https://github.com/MetaMask/metamask-extension/blob/develop/test/e2e/helpers.js#L549